content/topical/constrained_projections.md

```diff
@@ -11,7 +11,7 @@ doi: https://doi.org/10.5281/zenodo.5645153
 ---
 
 Outputs from the probabilistic projection methods developed or assessed in the
-European Climate Projection system (EUCP) Horizon2020 project. The data can be
+European Climate Projection system (EUCP) Horizon2020 project. <!--more-->The data can be
```

Comment:
Perhaps put the excerpt tag on a new line so it stands out a bit more when editing the markdown?

Reply:
Putting the excerpt tag on a new line will break the sentences into two paragraphs since the docs are rendered as html doc concerning the format. But it reads ok. I changed them.